easybuild/easyconfigs/n/NRGLjubljana/NRGLjubljana-2.4.3.23-foss-2016b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'NRGLjubljana'
+version = '2.4.3.23'
+
+homepage = 'http://nrgljubljana.ijs.si/'
+description = """
+  NRG Ljubljana is an efficient implementation of the numerical renormalization group (NRG)
+  technique for solving quantum impurity problems that arise as simplified models of
+  magnetic impurities and as effective models in the dynamical mean field theory (DMFT)
+  approach to bulk correlated materials.
+"""
+
+docurls = 'http://nrgljubljana.ijs.si/examples/'
+software_license = 'LicenseGPLv3'
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+#toolchainopts = { 'usempi': True, 'pic': True }
+toolchainopts = { 'pic': True }
+
+sources = [ '%(namelower)s-%(version)s.tar.gz' ]
+source_urls = [ 'http://nrgljubljana.ijs.si/' ]
+
+configopts = "--with-tools --with-boost-serialization=boost_serialization CXXFLAGS=\"-Ddsyev=dsyev_ -Ddsyevr=dsyevr_ -Dzheev=zheev_ -Dzheevr=zheevr_\" LDFLAGS=\"-lopenblas -L${EBROOTBOOST}/lib\""
```

Comment:
@rokzitko we maintain a max line lenght of 120 characters, so please break this up; also, use `'` as outer quotes (or `'` inside the string) rather than escaping `"`

```python
configopts = "..."
configopts += " ..."
```